wireapp/wire-android-reloaded
# feat: Create Incoming Call UI

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

There was no UI for incoming call

### Solutions

Incoming Call UI was implemented.

### Dependencies (Optional)

This is only the FiRST PR, more will follow soon, this is just the UI, no logic no nothing, currently the only way to open this screen is to click to start a call in a conversation (this will be changed later on as the logic is implemented in the following PRs)

Needs releases with:


#### How to Test
- Open the App
- Open a conversation
- Click on start a call/camera button at the top
- Screen should open and you can check the design


### Attachments
<img src="https://user-images.githubusercontent.com/5890660/163351247-f901f5d4-1e52-4eb8-ad53-b216447ed0f2.png" width="200" height="400" alt="incoming call ui" />
